### ISSUE TITLE
Can't see topology

### ISSUE THREAD

**Issue by gojkog21** [MurphyMc/poxdesk#5]
Hi,

I've installed POXDesk using the instructions you gave, but when I enter http://127.0.0.1:8000/poxdesk, I can't see topology in the appropriate box.

Thanks

**Comment by MurphyMc** [comment]
What commandline did you use to invoke POX?  And, importantly, did you include the openflow.discovery and tinytopo components?

**Comment by gojkog21** [comment]
I've used **./pox.py samples.pretty_log web messenger messenger.log_service messenger.ajax_transport openflow.of_service poxdesk** command.

I din't include any other component. 

How to invoke it??

Thanks

**Comment by MurphyMc** [comment]
The Getting Started page has this to say:
> Note: You might also want to run openflow.discovery and poxdesk.tinytopo to get the topology viewer working. You might want to run poxdesk.terminal to get the terminal working (won't work in Windows).
>
>Note: If you want to use POXDesk's learning switches (L2LS), you should not run a controller-side forwarding component (e.g., forwarding.l2_learning).

TL;DR: Add "openflow.discovery poxdesk.tinytopo" to your commandline to get the topology viewer working.

**Comment by gojkog21** [comment]
OK. Thanks!

**Comment by kirazero17** [comment]
I did include those two components, yet still no topology.
I ran the version from the tcp branch, from pox halosaur, with qooxdoo 3.5.
